### DISCUSSION TITLE
Running dbldatagen on EMR

### DISCUSSION THREAD

**Discussion by Nurkoo** [databrickslabs/dbldatagen#247] [Q&A] [answered]
Does dbldatagen work well when run as a PySpark application submitted to an EMR cluster?

**Comment by ronanstokes-db** [comment, marked as answer]
Its designed to be used as a library installed in a Databricks runtime environment. Its not licensed for use on an EMR cluster.

However, we dont do anything from a technical standpoint to block execution on a particular environment
